locale/en/knowledge/advanced/streams/what-are-streams.md

```diff
@@ -12,7 +12,7 @@ Streams are another basic construct in Node.js that encourages asynchronous codi
 
 In other words, Streams use events to deal with data as it happens, rather than only with a callback at the end. Readable streams emit the event `data` for each chunk of data that comes in, and an `end` event, which is emitted when there is no more data. Writeable streams can be written to with the `write()` function, and closed with the `end()` function. All types of streams emit `error` events when errors arise.
 
-As a quick example, we can write a simple version of `cp` (the Unix utility that copies files). We could do that by reading the whole file with standard filesystem calls and then writing it out to a file. Unfortunately, that requires that the whole file be read in before it can be written. In the case of 1-2 giga files, you could run into out of memory operations. The biggest advantage that streams give you over their non-stream versions are that you can start process the info before you have all the information. In this case, writing out the file doesn't get sped up, but if we were streaming over the internet or doing cpu processing on it then there could be measurable performance improvements.
+As a quick example, we can write a simple version of `cp` (the Unix utility that copies files). We could do that by reading the whole file with standard filesystem calls and then writing it out to a file. Unfortunately, that requires that the whole file be read in before it can be written. In the case of 1-2 giga files, you could run into out of memory operations. The biggest advantage that streams give you over their non-stream versions is that you can start process the info before you have all the information. In this case, writing out the file doesn't get speed up, but if we were streaming over the internet or doing cpu processing on it, then there could be measurable performance improvements.
```

Comment:
Previous text was better

Reply:
Nothing but change from "get speed up" instead of "get sped up", maybe a typo mistake?

Reply:
`sped` is correct here. I agree that the previous text was better. Maybe simplifying the entire sentence would be even better?:

> In this case, writing the files isn't faster, but if we were streaming over a network or doing CPU processing on the data, then there could be measurable performance improvements.

Reply:
OK, thank you Trott.